tap_rakuten/client.py

```diff
@@ -360,9 +360,10 @@ def get_schema(self, report_slug):
 
         logger.info("{} : determining schema.".format(report_slug))
 
-        future_date = datetime.now() + timedelta(days=2)
+        # 2023-09-26: API doesn't seem to accept future dates anymore
+        today = datetime.now()
 
-        with self.get(report_slug, start_date=future_date) as r:
+        with self.get(report_slug, start_date=today) as r:
             for line in r.iter_lines(decode_unicode=True, chunk_size=10):
```

Comment:
Will this possibly read column values as separate columns? Why do we even need to iterate through lines? Couldn't we just process the first line (which I'm assuming is the headers)? I'm making some assumptions about the structure of the report, but the comment about "minimal rows" above got me thinking about what the potential downside might be to including any rows accidentally.

Reply:
Good point. Looking at the rest though, it looks like it `breaks` after the first line - 

```
for line in r.iter_lines(decode_unicode=True, chunk_size=10):
  columns = line.split(",")
  break
```
Also tested this change locally and nothing broke. But I guess I could change it to 
```
with self.get(report_slug, start_date=today) as r:
  columns = r.iter_lines(decode_unicode=True, chunk_size=10)[0].split(",")
```
do you think that's a better move?

Reply:
huh, good point about the `break`. Yeah I was thinking something like just accessing the first line. You can use the `next()` function instead of `[0]` to get the first line off the iterator, like:
```
iterator = r.iter_lines(decode_unicode=True, chunk_size=10)
first_line = next(iterator)
columns = first_line.split(",")
```
(https://tedboy.github.io/requests/adv13.html?highlight=next)
I'm not sure what the benefits are of `next` vs `[0]` index, but 🤷

Reply:
Like you said, the current code accomplishes the same thing, but maybe it's clearer for the future to omit the loop

Reply:
ah right! thanks, that definitely feels cleaner, made the change - mind taking a last look?